examples/demo/src/reviews/ReviewCreate.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+    SimpleForm,
+    Create,
+    ReferenceInput,
+    TextInput,
+    DateInput,
+    AutocompleteInput,
+    required,
+    useNotify,
+    useRedirect,
+    getRecordFromLocation,
+} from 'react-admin';
+
+import StarRatingInput from './StarRatingInput';
+import { useLocation } from 'react-router';
+
+const ReviewCreate = () => {
+    const notify = useNotify();
+    const redirect = useRedirect();
+    const location = useLocation();
+
+    const onSuccess = (_: any) => {
+        const record = getRecordFromLocation(location);
```

Comment:
It's not a product, it's an object containing default values for the new review. I would leave it as it is or rename it to `formInitialValues`

Reply:
If the parameter name is _,  it is due to a lint warning unused parameter. I prefer to let it like that. It is the review but from this parameter I have no clue to know if the review comes from a simple creation or a creation from a product.

Reply:
@fzaninotto and I were referring to the `record` variable ;)